jdaviz/configs/default/plugins/data_tools/data_tools.vue

```diff
@@ -3,7 +3,7 @@
     <v-dialog v-model="dialog" height="400" width="600">
       <template v-slot:activator="{ on }">
         <v-btn tile depressed v-on="on" color="turquoise">
-          Import Data
+          <v-icon>mdi-plus</v-icon>
```

Comment:
What about this? I saw this when I filter with "file" in https://materialdesignicons.com/

```suggestion
          <v-icon>file-document-plus-outline</v-icon>
```

Reply:
There is also `file-image-plus-outline`.